src/browser/base/content/zen-assets.jar.inc.mn

```diff
@@ -52,6 +52,8 @@
         content/browser/zen-styles/zen-gradient-generator.css                   (../../zen/workspaces/zen-gradient-generator.css)
 
         content/browser/zen-components/ZenKeyboardShortcuts.mjs                 (../../zen/kbs/ZenKeyboardShortcuts.mjs)
+        content/browser/zen/modules/ZenAirTrafficControl.mjs                    (../../zen/modules/ZenAirTrafficControl.mjs)
```

Comment:
Please stick to having all of the JS related modules inside `content/browser/zen-components`